module/cloud-services/files/validator/v1.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * name : cloud-services/files/validator/v1.js
+ * author : Saish
+ * created-date : 07-JUNE-2024
+ * Description : Files validator.
+ */
+
+module.exports = (req) => {
```

Comment:
@borkarsaish65 do we need this validation? we are adding endpoints in files controller right? why we need validation in multiple files for same endpoint?

Reply:
@VISHNUDAS-tunerlabs I have removed the dead code now.